library/kotlin/src/io/envoyproxy/envoymobile/PulseClient.kt

```diff
@@ -13,6 +13,8 @@ interface PulseClient {
    */
   fun counter(vararg elements: Element): Counter
 
+  fun counter(vararg elements: Element, tags: List<Tag>): Counter
```

Comment:
i imagine this api will be used to add tags during(pre) compile time.

caveat on kotlin: kotlin does not allow two vararg parameters for methods, hence `List`.

for swift the [current api](https://github.com/envoyproxy/envoy-mobile/blob/fe12371436065d52da1a3807956d6dbd17d3fd63/library/swift/src/PulseClient.swift#L12) uses an array for elements